pkgs/development/ocaml-modules/ocaml-lsp/jsonrpc.nix

```diff
@@ -10,10 +10,14 @@
 }:
 
 let params =
-  if lib.versionAtLeast ocaml.version "4.13"
+  if lib.versionAtLeast ocaml.version "4.14"
   then {
-    version = "1.10.3";
-    sha256 = "sha256-o6wQc7Byi5T0vbARF3LAq69/9wMkOZRQ6rcVa/rBUfE=";
+    version = "1.11.1";
+    sha256 = "sha256-jqiqGB6wXSXtWyqOAS/zZNPa0ezHthQdeLtYI0gc1qQ=";
+  } else if lib.versionAtLeast ocaml.version "4.13"
+  then {
+    version = "1.10.5";
```

Comment:
```suggestion
    version = "1.10.6";
```
1.10.6 is released

Reply:
1.10.6 silently dropped support for ocaml 4.13 and only targets 4.14. It's not nice for a patch release.